indico/modules/categories/controllers/display.py

```diff
@@ -612,9 +617,14 @@ def _find_nearest_category(self, category_chain):
         raise Exception(f'Category {self.category.id} not found in category chain')
 
     def _get_event_data(self, event_query):
+        def calculate_keyword_id(kw):
+            return int(hashlib.sha256(kw.encode()).hexdigest()[:8], 16) + 3
```

Comment:
What about a simple `crc32` hash instead? IIRC we event have a util for it somewhere.

Reply:
Yep, sounds good.